### PR TITLE
[SD-1988] Closing output channels on error

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-publish/views/output-channels.html [hunk 1]
```diff
@@ -29,73 +29,93 @@ <h3 translate ng-hide="outputChannel._id" translate>Add New Output Channel</h3>
...
+                    </tab>
+                    
+
+                    <tab heading="Critical Errors" class="TabContent">
+                        <div class="content privileges-settings">
+                            <div class="horizontal">
+                                <div class="roles-list">
+                                    <div ng-repeat="(key, val) in all_errors | orderBy:name" class="field">
+                                        <label>
+                                            {{key}}: {{val}}
```
Comment (plamut) [inline client/app/scripts/superdesk-publish/views/output-channels.html]: Are these two values fixed (i.e. remain unchanged)? If so, we should use one-time bindings for them.

File: client/app/scripts/superdesk-publish/views/subscribers.html [hunk 2]
```diff
@@ -29,37 +29,56 @@ <h3 translate ng-hide="subscriber._id" translate>Add New Subscriber</h3>
...
+                        </div>
+                    </tab>
+
+                    <tab heading="Critical Errors" class="TabContent">
+                        <div class="content privileges-settings">
+                            <div class="horizontal">
+                                <div class="roles-list">
+                                    <div ng-repeat="(key, val) in all_errors | orderBy:name" class="field">
+                                        <label>
+                                            {{key}}: {{val}}
```
Comment (plamut) [inline client/app/scripts/superdesk-publish/views/subscribers.html]: Same comments as above (i.e. possible to use one-time bindings?).

File: client/app/scripts/superdesk-publish/views/output-channels.html (hunk 1 above)
Comment (akintolga) [inline client/app/scripts/superdesk-publish/views/output-channels.html]: You are quite right Peter. I'm fixing those. Cheers.

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by plamut** [comment]
:+1:

**Pull request merged**